.github/CODEOWNERS

```diff
@@ -4,7 +4,9 @@
 # For syntax help see:
 # https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
 
-*                       @googleapis/yoshi-java
+# The @googleapis/api-bigtable is the default owner for changes in this repo
+*                       @googleapis/yoshi-java @googleapis/api-bigtable
```

Comment:
can you add this to .repo-metadata.json, this file is generated

Reply:
oops yes, added and reverted the exclusion for the CODEOWNERS file